var/spack/repos/builtin/packages/albany/package.py

```diff
@@ -129,8 +142,22 @@ def cmake_args(self):
 
         if "+sfad" in spec:
           options.extend([
+            "-DENABLE_FAD_TYPE:STRING=SFad",
             "-DALBANY_SFAD_SIZE=%d" % int(spec.variants["sfadsize"].value)
                        ])
+        elif "+slfad" in spec:
+          options.extend([
+            "-DENABLE_FAD_TYPE:STRING=SLFad",
+            "-DENABLE_TAN_FAD_TYPE:STRING=SLFad",
+            "-DENABLE_HES_VEC_FAD_TYPE:STRING=SLFad",
+            "-DALBANY_SLFAD_SIZE=90",
+            "-DALBANY_TAN_SLFAD_SIZE=100",
+            "-DALBANY_HES_VEC_SLFAD_SIZE=100"
+                       ])
```

Comment:
Note: this is needed for Albany tests but it might not be needed for mali/e3sm tests. in fact, I think one of the e3sm tests fails with this because of oom issues. sfad12 might be sufficient for all mali/e3sm tests.